.github/workflows/swift-test.yml

```diff
@@ -14,8 +14,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-      with:
-        swift-version: '5.2.0'
-
+    - name: Download and Run  MeiliSearch
+      run:  brew update && brew install meilisearch
+    - name: Run MeiliSearch
+      run: meilisearch --no-analytics=true &
     - name: Run tests
       run: swift test -v
```

Comment:
Why don't we use the script file for tests here?

Reply:
The script file also runs a docker. I think pedro made it to ease the testing when developing.